jxl/src/util/test.rs

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) the JPEG XL Project Authors. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+#[macro_export]
```

Comment:
I think `pub use assert_almost_eq` and then importing `util::test::assert_almost_eq` would be better

Reply:
using `pub(crate)...` now